examples/teleportation.py

```diff
@@ -29,7 +29,7 @@
     Squeezed(2) | q[2]
 
     # apply gates
-    BS = BSgate(pi / 4, pi)
+    BS = BSgate(pi / 4,0)
```

Comment:
Just to conform to PEP 8.

```suggestion
    BS = BSgate(pi / 4, 0)
```